.github/labeler-pr.yml

```diff
@@ -1,20 +1,20 @@
-repo:
+"scope: repo":
   - "*"
 
-ci:
+"scope: ci":
   - ".github/**/*"
 
-applications:
+"scope: applications":
```

Comment:
@brlipi perhaps sort labels alphabetically for easier maintenance and readability?

Reply:
Labels rearranged.

Reply:
@brlipi I stlll don't see any changes, and no new commits here https://github.com/devopness/devopness/pull/838/commits

Reply:
See commit https://github.com/devopness/devopness/pull/838/commits/b6ae1cf50eb69b3fa9987f9655c2c4217375117a